sources/procfs.go

```diff
@@ -72,6 +75,11 @@ type lustreHelpStruct struct {
 	helpText string
 }
 
+type statsHelpStruct struct {
```

Comment:
@joehandzik I'm not crazy about this name for obvious reasons. Any suggestion on what might be better?